src/jit/gentree.h

```diff
@@ -1916,6 +1916,9 @@ class GenTreeUseEdgeIterator final
 #ifdef FEATURE_SIMD
     void MoveToNextSIMDUseEdge();
```

Comment:
The following method probably should not have a default parameter:

 `IteratorPair<GenTreeUseEdgeIterator> GenTree::UseEdges(bool expandMultiRegArgs = true);
`